apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -32,6 +32,9 @@ const (
 
 	// DefaultUnavailablePeriodSeconds is the default period of time we consider a newly applied workload as unavailable.
 	DefaultUnavailablePeriodSeconds = 60
+
+	// DefaultMaxSkewValue is the default degree to which resources may be unevenly distributed.
+	DefaultMaxSkewValue = 1
```

Comment:
can you move all the default value to the default package? i think they're not related to the api version.

defaulter package makes more sense